src/Sql/Sql.php

```diff
@@ -148,6 +148,54 @@ public function delete($table = null)
     }
 
     /**
+     * @param null|string|array|TableIdentifier $table
+     * @return Ddl\AlterTable
+     * @throws Exception\InvalidArgumentException
+     */
+    public function alterTable($table = null)
```

Comment:
This is not needed, because there is already a [DDL Abstraction](https://docs.zendframework.com/zend-db/sql-ddl/).